slsDetectorSoftware/src/Detector.cpp

```diff
@@ -405,6 +454,104 @@ Result<int> Detector::getTemperature(defs::dacIndex index,
     return res;
 }
 
+std::vector<defs::dacIndex> Detector::getDacList() const {
+    std::vector<defs::dacIndex> retval;
+    int i = 0;
```

Comment:
I is not used until the for loop so it can be moved into the loop declaration

Reply:
done